cmd/podman/common/build.go

```diff
@@ -116,6 +116,14 @@ func DefineBuildFlags(cmd *cobra.Command, buildOpts *BuildFlagsWrapper) {
 		_ = flags.MarkHidden("logsplit")
 		_ = flags.MarkHidden("cw")
 	}
+	if isFarmBuild {
+		_ = flags.MarkHidden("all-platforms")
+		_ = flags.MarkHidden("arch")
+		_ = flags.MarkHidden("platform")
+		_ = flags.MarkHidden("stdin")
+		_ = flags.MarkHidden("variant")
+		_ = flags.MarkHidden("manifest")
```

Comment:
Are these options rejected when specified?  If not, what happens when people use them?

Reply:
I added a check for these as well to throw an error when set.